Update upgrade-aks-cluster.md

ramdaspotale opened the pull request:
Update upgrade-aks-cluster.md to fix --drain-timeout parameter on az aks nodepool update. 

according to https://learn.microsoft.com/en-us/cli/azure/aks/nodepool?view=azure-cli-latest#az-aks-nodepool-update and also from my experience correct parameter to set drain timeout is --drain-timeout and not --drainTimeoutInMinutes.

ramdaspotale committed:
Update upgrade-aks-cluster.md

Update upgrade-aks-cluster.md to fix --drain-timeout parameter on az aks nodepool update. 

according to https://learn.microsoft.com/en-us/cli/azure/aks/nodepool?view=azure-cli-latest#az-aks-nodepool-update and also from my experience correct parameter to set drain timeout is --drain-timeout and not --drainTimeoutInMinutes.

ramdaspotale committed:
Merge branch 'main' into patch-1

schaffererin commented on articles/aks/upgrade-aks-cluster.md:
```diff
@@ -244,7 +244,7 @@ At times, you may have a long running workload on a certain pod and it can't be
+    az aks nodepool add -n mynodepool -g MyResourceGroup --cluster-name MyManagedCluster   --drain-timeout 100
 
     # Update drain timeout for an existing node pool
     az aks nodepool update -n mynodepool -g MyResourceGroup --cluster-name MyManagedCluster --drain-timeout 45
```
```suggestion
    az aks nodepool add --name mynodepool --resource-group myResourceGroup --cluster-name myManagedCluster --drain-timeout 100

    # Update drain timeout for an existing node pool
    az aks nodepool update --name mynodepool --resource-group myResourceGroup --cluster-name myManagedCluster --drain-timeout 45
```

schaffererin reviewed (changes requested):
@ramdaspotale, thank you for your contributions to the AKS docs! I left feedback for you to address, otherwise, the PR is good for sign off. Please let me know if you have any questions. :)

ramdaspotale committed:
Update articles/aks/upgrade-aks-cluster.md

schaffererin replaced short parameter flags to full flag keywords. its good which i missed it.

Co-authored-by: Erin Schaffer <69220686+schaffererin@users.noreply.github.com>
